knowledge-base/window-backdrop-blur.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Apply a backdrop filter with a blur effect to the Modal Window background
+description: How to apply a backdrop filter with a blur effect to the background content of the Modal Window.
+type: how-to
+page_title: Apply a backdrop filter with a blur effect to the Modal Window background
+slug: window-kb-backdrop-blur
+position: 
+tags: 
+ticketid: 1642061
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Window for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+* How do I blur the background in Windows?
```

Comment:
Do you mean the operating system or the component? Try using precise expressions.